### DISCUSSION TITLE
foobar2000 使用指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#25] [Announcements]
# foobar2000 使用指南

foobar2000 可以说是 Windows 平台相当好用的全能型音频相关软件了，集播放、转码、信息编辑等各种功能于一身，且图形化、上手简单。将日期设置为 2019-07-16 是为了纪念本文的初成。

https://zexwoo.blog/posts/tutorials/foobar2000/

**Comment by BreadKi11er** [comment]
你好，请问右键选中音轨后的「收集」一项有什么作用？点按之后会移除其它所有的音轨？
